exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -361,16 +414,14 @@ func (ku *Kucoin) UpdateTicker(ctx context.Context, p currency.Pair, assetType a
 
 // UpdateTickers updates all currency pairs of a given asset type
 func (ku *Kucoin) UpdateTickers(ctx context.Context, assetType asset.Item) error {
+	var errs error
 	switch assetType {
 	case asset.Futures:
 		ticks, err := ku.GetFuturesOpenContracts(ctx)
 		if err != nil {
 			return err
 		}
 		pairs, err := ku.GetEnabledPairs(asset.Futures)
```

Comment:
There's a linter issue / static check issue since the err here is never checked

Reply:
fixed the linter issues